docs/sdk/src/guides/your_first_pallet.rs

```diff
@@ -0,0 +1,325 @@
+//! # Currency Pallet
+//!
+//! By the end of this guide, you will write a small FRAME pallet (see
+//! [`crate::polkadot_sdk::frame_runtime`]) that is capable of handling a simple crypto-currency.
+//! This pallet will:
+//!
+//! 1. Allow anyone to mint new tokens into accounts (which is obviously not a great idea for a real
+//!    system).
+//! 2. Allow any user that owns tokens to transfer them to others.
+//! 3. Track the total issuance of all tokens at all times.
+//!
+//! > This guide will build a currency pallet from scratch using only the lowest primitives of
+//! > FRAME, and is mainly intended for education, not *applicability*. For example, almost all
+//! > FRAME-based runtimes use various techniques to re-use a currency pallet ([`pallet_balances`]
+//! > or [`pallet_assets`] to be precise) instead of writing one. Further advanced FRAME related
+//! > topics are discussed in [`crate::reference_docs`].
+//!
+//! ## Topics Covered
+//!
+//! The following FRAME topics are covered in this guide:
+//!
+//! - [Storage](frame::pallet_macros::storage)
+//! - [Call](frame::pallet_macros::call)
+//! - [Event](frame::pallet_macros::event)
+//! - [Error](frame::pallet_macros::error)
+//! - Basics of testing a pallet
+//! - [Constructing a runtime](frame::runtime::prelude::construct_runtime)
+// TODO: fix links to pallet macros
```

Comment:
Flagging similar to earlier comment, should check the dest. of these links is to the rich docs rather than the stub. Or, maybe move the rich docs to `frame` and replace `frame_support` usage with it throughout the codebase...

```suggestion
//! - [Storage](frame_support::pallet_macros::storage)
//! - [Call](frame_support::pallet_macros::call)
//! - [Event](frame_support::pallet_macros::event)
//! - [Error](frame_support::pallet_macros::error)
//! - Basics of testing a pallet
//! - [Constructing a runtime](frame_support::runtime::prelude::construct_runtime)
// TODO: fix links to pallet macros
```